### PR TITLE
Paragraph updater db

### PR DESCRIPTION
**Pull request by elvinukas**
Paragraph is now not stored in the StatisticsModel class itself, only its key. This is to ensure that paragraph is only retrieved if it is needed.
Game class still holds StatisticsModel within itself (meaning it stores not only its id, but also its data), since Game class methods mainly compose of manipulating with Statistics data to retrieve final results of Statistics and Game properties.
Perhaps could be changed in the future, but for now lets focus on other work.

### CODE REVIEW

File: Typeracer/Controllers/GraphController.cs [hunk 1]
```diff
@@ -17,27 +13,44 @@
...
+
+    public GraphController(AppDbContext context)
+    {
+        _dbContext = context;
+    }
+    
+    
     [HttpPost("generate")]
-    public IActionResult GenerateGraph([FromBody] string gameId, AppDbContext context)
+    public IActionResult GenerateGraph([FromBody] string gameId)
```
Comment (UPovilas) [inline Typeracer/Controllers/GraphController.cs:27]: I changed the controller to handle only HTTP requests in my branch... Conflicts incoming!

File: Typeracer/Controllers/HomeController.cs [hunk 2]
```diff
@@ -1,20 +1,42 @@
...
+        try
+        {
+            InsertParagraphFileToDb("paragraph1.txt", simpleGamemodes);
+
+            InsertParagraphFileToDb("paragraph2.txt", shortGamemode);
+            
+        }
+        catch (NoAddToDBException e)
+        {
+            Console.WriteLine(e.Message, "Intentional? - " + e.Intentional);
```
Comment (UPovilas) [inline Typeracer/Controllers/HomeController.cs:37]: Maybe we should change Console.Writeline to ILogger everywhere?

**Review by UPovilas — APPROVED** [review]
Overall great job! Graph Controller may have to be changed due to the fact that I switched method of generating graph to a separate file

**Review by RytasV — APPROVED** [review]
All G

File: Typeracer/Controllers/HomeController.cs (hunk 2 above)
Comment (elvinukas) [reply Typeracer/Controllers/HomeController.cs:37]: yeah, sorry for not replying earlier! yes, changing it is the best option

**Pull request merged**